Invoice.py

```diff
@@ -160,6 +161,25 @@ def search_tokens(self, text: str, token_list="group"):
 
         return filtered_tokens
 
+    def remove_stopwords(self):
```

Comment:
Could remove unless there is a use within the InvoicePage object. This function was moved over to the OCREngine so that it can be used as part of the OCR process

Reply:
i'll comment it out for now